pkg/bgpv1/manager/reconcile_test.go

```diff
@@ -50,8 +50,8 @@ func TestPreflightReconciler(t *testing.T) {
 			name:           "no change",
 			routerID:       "192.168.0.1",
 			newRouterID:    "192.168.0.1",
-			localPort:      45450,
-			newLocalPort:   45450,
+			localPort:      30450,
```

Comment:
Note that this is still not bullet-proof, as this range is configurable. The other option would be to make affected tests privileged and use common ports `< 1024`

Reply:
Filed a follow-up in https://github.com/cilium/cilium/issues/26209